appeals/web/src/server/lib/mappers/appeal.mapper.js

```diff
@@ -910,5 +912,52 @@ export async function initialiseAndMapAppealData(data, currentRoute, session) {
 		}
 	};
 
+	/** @type {Instructions} */
+	mappedData.appeal.issueDeterminationDate = {
+		display: {
+			summaryListItem: {
+				key: {
+					text: 'Issue Determination'
+				},
+				value: {
+					html:
+						dateToDisplayDate(data.appeal.appealTimetable?.issueDeterminationDate) ||
```

Comment:
It would be good to add one or more unit tests which render a date into the snapshots rather than just the fallback text. I noticed while fixing a date-related bug yesterday that `dateToDisplayDate` was returning the incorrect date (it seems to be using the US format MM/DD/YYYY in some circumstances, could just have been down to me using it incorrectly) so would be good to confirm that's not happening here.

Reply:
FYI I ended up using `webDateToDisplayDate` instead which doesn't exhibit the same problem